tests/Type/EnumTypeTest.php

```diff
@@ -348,8 +348,7 @@ public function testDoesNotAcceptValuesWithIncorrectCasing(): void
             '{ colorEnum(fromEnum: green) }',
             null,
             [
-                // Improves upon the reference implementation
```

Comment:
Too bad we lost that improvement, but it does not matter too much. Ease of maintenance trumps this IMO.